chaosk8s/actions.py

```diff
@@ -143,3 +144,21 @@ def scale_microservice(name: str, replicas: int, ns: str = "default",
         raise ActivityFailed(
             "failed to scale '{s}' to {r} replicas: {e}".format(
                 s=name, r=replicas, e=str(e)))
+
+
+def scale_statefulset(name: str, replicas: int, ns: str = "default",
```

Comment:
should this belong to a different PR since this PR is about create_service?

Reply:
I will wait for https://github.com/chaostoolkit/chaostoolkit-kubernetes/pull/70 approval before creating a new PR. I've made several updates from the same master branch, its a bit confusing
Nevertheless, you're right. I will remove this part from this PR

Reply:
https://github.com/chaostoolkit/chaostoolkit-kubernetes/pull/73